.travis.yml

```diff
@@ -45,7 +46,10 @@ before_script:
 
 script:
     - make
-    - if [ -z "$CROSS_COMPILE" ]; then make test; fi
+    - if [ -n "$CROSS_COMPILE" ]; then
+          export EXE_PRE="wine " EXE_EXT=".exe" WINEPREFIX=`pwd`;
```

Comment:
As for failure without EXE_PRE. That's what I get too, in real non-container environment. And that's what I meant when originally said "doesn't work *right now* [but should be possible to fix]". I mean the failure is not necessarily indication of incapability of a container to handle transparent execution of PE-COFF binaries. More to come...